pysite/constants.py

```diff
@@ -80,3 +80,9 @@ class ValidationTypes(Enum):
 # CSRF
 
 CSRF = CSRFProtect()
+
+# GitHub Token
+GITHUB_TOKEN = environ.get("GITHUB_TOKEN") or None
```

Comment:
Doesn't `environ.get()` return `None` by default?

Reply:
Since we set defaults in the rest of the file I just kept it that way